core/src/test/java/nl/nn/adapterframework/util/EnumUtilsTest.java

```diff
@@ -73,6 +78,30 @@ public void testParseNonExistingDocumentedEnumWithCustomFieldName() {
 		EnumUtils.parseDocumented(FtpType.class, "fieldName", "tralala");
 	}
 
+	public static enum TestDocumentedEnum implements DocumentedEnum { @EnumLabel("een") ONE, @EnumLabel("twee") TWO; }
+	public static enum TestNotDocumentedEnum { ONE, TWO; }
```

Comment:
Misschien niet voor in deze pullrequest: zou @EnumLabel niet ook zonder 'DocumentedEnum' kunnen?